inst/include/mode/mode.hpp

```diff
@@ -63,6 +58,18 @@ class dust_ode {
     return m_.n_variables();
   }
 
+  size_t n_pars() const {
+    return 0;
```

Comment:
I'm a little confused by this, why is `n_pars` fixed to 0?

Reply:
n_pars should be 0 unless the object was created in "multiple parameter mode", which is not supported yet in mode. I've added a comment here, see https://mrc-ide.github.io/dust/reference/dust_generator.html#method-n-pars- and https://github.com/mrc-ide/dust/blob/master/inst/include/dust/dust_cpu.hpp#L44 for details.

One reason why this is probably confusing is that the implementation for `n_pars_effective` was wrong and should return 1